provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -5,26 +5,20 @@ name: iPXE intermediate script
 snippet: false
 -%>
 #!ipxe
+# Intermediate iPXE script to report MAC address to Foreman
 
-# Skips booting from network and continues booting from next device
-exit
+<% (0..32).each do |i| -%>
+:net<%= i %>
+isset ${net<%= i -%>/mac} || goto no_nic
+dhcp net<%= i -%> || goto net<%= i+1 %>
+chain <%= foreman_url('iPXE') %>?mac=${net0/mac} || goto net<%= i+1 %>
```

Comment:
Looks like i was to quick to merge, should this be:
```
 chain <%= foreman_url('iPXE') %>?mac=${net<%= i -$>/mac} || goto net<%= i+1 %>
```
?

Reply:
Yes I can confirm.